bundles/org.openhab.binding.plclogo/README.md

```diff
@@ -81,10 +80,10 @@ Thing analog <ThingId>  "Label" @ "Location" [ kind="<kind>", threshold=<number>
 | Parameter | Type    | Required   | Default   | Description                                                   |
 | --------- | :-----: | :--------: | :-------: | ------------------------------------------------------------- |
 | kind      | String  | Yes        |           | Blocks kind                                                   |
-| threshold | Integer | No         | 0         | Send current value to openHAB, if changed more than threshold |
-| force     | Boolean | No         | false     | Send current value to openHAB, independent if changed or not  |
+| threshold | Integer | No         | 0         | Send current value to openHAB if changed by more than threshold |
+| force     | Boolean | No         | false     | Send current value to openHAB, independent of whether changed or not  |
```

Comment:
It would be nice if this table could be reformatted as it was previously nicely aligned.